packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, number } from '@storybook/addon-knobs';
+import YoutubePlayerTestComponent from './YoutubePlayer';
+
+storiesOf('Sandbox', module)
```

Comment:
adding this is great!

Reply:
let's create a new sandbox section just for `audio players!

Reply:
> let's create a new sandbox section just for `audio players!

okay working on it